File: src/main/java/com/movisens/xs/api/exceptions/AuthorizationException.java
package com.movisens.xs.api.exceptions;

import com.movisens.xs.api.models.ApiError;


public class AuthorizationException extends MovisensXSException {

    private static final long serialVersionUID = -1523752187360750888L;

    public AuthorizationException(ApiError apiError) {
        super(apiError);
    }
}

File: src/main/java/com/movisens/xs/api/exceptions/AuthorizationException.kt
package com.movisens.xs.api.exceptions

import com.movisens.xs.api.models.ApiError

class AuthorizationException(apiError: ApiError?) : MovisensXSException(apiError) {
    companion object {
        private const val serialVersionUID = -1523752187360750888L
    }
}